examples/defaulter-gen/generators/defaulter.go

```diff
@@ -466,9 +464,7 @@ func getNestedDefault(t *types.Type) string {
 		if len(defaultMap) == 1 && defaultMap[0] != "" {
 			return defaultMap[0]
 		}
-		if t.Kind == types.Alias {
-			t = t.Underlying
-		} else if t.Kind == types.Pointer {
+		if t.Kind == types.Pointer {
```

Comment:
Changing this logic seems to remove the ability for a default attached to an alias's underlying to be used. Is that correct?

Reply:
I want to argue that this FEELS correct (the same way that `type Y X` - Y doesn't inherit X methods, it shouldn't inherit defaults), but I don't know if that would break anything in k/k without actually trying.